Fabric.Authorization.API/Modules/EdwAdminModule.cs

```diff
@@ -72,23 +74,34 @@ private async Task<dynamic> SyncGroupIdentityRoles(dynamic param)
         {
             CheckInternalAccess();
 
+            var groupIdentifier = new GroupIdentifier
+            {
+                GroupName = param.groupName,
+                IdentityProvider = SetIdentityProvider(param.identityProvider),
+                TenantId =  param.tenantId

```

Comment:
Yes, there are cases where IdP and TenantId will be null, whether because of existing clients not including that info in the request or in the case of AD, they will never have a TenantId. For the case where clients don't post the info in the request when creating the group, we default to the IdP specified in the config file.